core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -679,4 +813,30 @@ private ExprValue exprYear(ExprValue date) {
     return new ExprIntegerValue(date.dateValue().getYear());
   }
 
+  /**
+   * isValidMySqlTimeZoneId for timezones which match timezone the range set by MySQL.
+   *
+   * @param zone ZoneId of ZoneId type.
+   * @return Boolean.
+   */
+  private Boolean isValidMySqlTimeZoneId(ZoneId zone) {
+    ZoneId maxTz = ZoneId.of("+14:00");
```

Comment:
"+14:00" is a magic number and should be defined as a constant at the top of the time. 
You can re-use that constant in your tests.  
Same for "-13:59" and "+00:00"